microcosm_postgres/context.py

```diff
@@ -10,30 +10,38 @@
 from microcosm_postgres.operations import new_session, recreate_all
 
 
-class SessionContext:
-    """
-    Save current session in well-known location and provide context management.
+class SessionContextFactory:
```

Comment:
We now make `SessionContext` an instance of `SessionContextFactory` to add in the `sesison` property